ntbs-service/Pages/Notifications/Edit/Episode.cshtml

```diff
@@ -0,0 +1,91 @@
+@page "{handler?}"
+@model ntbs_service.Pages_Notifications.EpisodeModel
+@using static NHSUK.FrontEndLibrary.TagHelpers.FormGroupType
+
+@{
+    ViewData["Title"] = "Notification - Hospital Details";
+}
+
+<h2>Hospital details</h1>
+
+<div class="row">
+    <div class="col-md-5">
+        <form method="post">
+            <input type="hidden" asp-for="NotificationId" />
+            <div asp-validation-summary="ModelOnly" class="text-danger"></div>
+
+            <cascading-dropdowns inline-template>
+                <div>
+                    <div class="nhsuk-form-group">

```

Comment:
We could use the [nhs razor tags](https://github.com/nhsuk/frontend-dotnetcore) here to streamline the markup a little?